templates-extension.md

```
@@ -739,20 +781,20 @@ quality: 3
 - Christian County, Kentucky, 1799 Tax Book; FHL microfilm 7,926.
 
-### Input Matching
+## Input Matching
 
 For applications that support the `alt*` fields, such fields can be used to assist in converting a non-templated source to a templated source by providing an appropriate default for a user to confirm.
-Let’s look at an example using the Web Database model from *Evidence Explained*, pages 254 and 438.
+Let's look at an example using the Web Database model from *Evidence Explained*, pages 254 and 438.
 
 We will use the following YAML snippet as an example:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)